internal/aggregator/utils.go

```diff
@@ -14,7 +15,7 @@ import (
 
 func extractJobSetAttrs(js *jobset.JobSet) records.Attrs {
```

Comment:
With this function now growing in complexity, lets add a table of unit tests.

Reply:
added in a2fa4ab